automatic/alldup/tools/chocolateyInstall.ps1

```diff
@@ -4,21 +4,24 @@ $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 
 $packageArgs = @{
   packageName    = $Env:ChocolateyPackageName
-  fileType       = $fileType
-  file           = Get-Item $toolsPath\*.exe
+  fileType       = 'EXE'
+  file           = Join-Path -Path $toolsPath -ChildPath 'AllDupSetup.exe'
```

Comment:
Why? They both do the same thing, and `Join-Path` is safer.

Reply:
> Then to a search replace in the update.ps1 file similar to how we do it in the calibre package (in case the executable name changes in the future).

👍 I'll look at that.

Reply:
Using `Join-Path` also makes it harder to do a search replace in the `update.ps1` file (which we very commonly do).

It also goes against the consistency we have of not using it (unless when necessary).